### ISSUE TITLE
using xhp

### ISSUE THREAD

**Issue by kennethrapp** [tengyifei/openshift-cartridge-nginx-hhvm#37]
I managed to get Composer to load Facebook's XHP library with the `--ignore-platform-reqs` flag, and I added the following line to config/hhvm.d/config.ini.erb

`hhvm.enable_xhp = true`

However, I still get the following error when I try to use XHP tags: 

`unknown class x_xhp__composable_element`

which appears to mean that XHP is not actually enabled?

**Comment by tengyifei** [comment]
Does it happen when you load any tags? Even trivial ones like `<span></span>`?

**Comment by kennethrapp** [comment]
yes, I'm not even trying to use custom tags. At the moment, this is my entire index.php

    <?hh 
    require_once("../vendor/autoload.php");
    $content=<div>hi</div>;
    echo $content;
